Sources/Build/BuildOperation.swift

```diff
@@ -461,43 +460,46 @@ public final class BuildOperation: PackageStructureDelegate, SPMBuildCore.BuildS
         // Compile the plugin, getting back a PluginCompilationResult.
         class Delegate: PluginScriptCompilerDelegate {
```

Comment:
does this have a subclass? Could become `final` if not
```suggestion
        final class Delegate: PluginScriptCompilerDelegate {
```

Reply:
Fixed!